recipes/hybsuite/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "hybsuite" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Yuxuanliu-HZAU/HybSuite/archive/v{{ version }}.tar.gz
+  # curl -sL https://github.com/Yuxuanliu-HZAU/HybSuite/archive/v1.1.0.tar.gz | sha256sum
+  sha256: db697c3520277eb391bb1d828ef11d076f0c34bf7e5270de1e178a27059346a0
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('hybsuite', max_pin="x.x") }}
+
+requirements:
+  build:
+    - python =3.9.15
+    - pip
+    - setuptools
+    - make
+  run:
+    - bash
+    - sra-tools
+    - pigz
+    - trimmomatic
+    - hybpiper
+    - amas
+    - mafft
+    - trimal
+    - modeltest-ng
+    - iqtree
+    - raxml
+    - raxml-ng
+    - newick_utils
+    - r-base
+    - r-phytools
+    - r-ape
+    - python =3.9.15
+    - pip
+    - paragone =1.1.1
+    - pyqt
+    - pyqt5-sip
+    - ete3
+    - pandas
+    - seaborn
+    - matplotlib-base
+    - numpy
+    - phykit
+
+test:
+  commands:
+    - test -f $PREFIX/bin/HybSuite.sh
+    - test -f $PREFIX/bin/hybsuite
```

Comment:
the standard test here is to call the tool (like the next line) which tests its existence as well as that it is executable and on the path, so these `test -f` lines are not needed.